pkg/build/api/types.go

```diff
@@ -86,6 +95,10 @@ type BuildStatus struct {
 	// Cancelled describes if a cancel event was triggered for the build.
 	Cancelled bool
 
+	// Skipped indicates that this build was marked as skipped. The build will be
```

Comment:
This should be a condition, not a boolean.  Add a conditions array, and we need to define a condition that represents "not yet ready to be scheduled" (it's similar to pending or waiting or queued).  We need to tighten up terminology - I don't know that I like `schedule` as the verb here because that already means something specific elsewhere.  On Jobs we use `Parallelism`, but do we really anticipate multiple policies?  Can you describe a 3rd policy?